.circleci/config.yml

```diff
@@ -97,24 +97,20 @@ jobs:
       - *restore_repo_cache
       - *install_tox
       - run: tox -e py27-unit
-  py27_docstring:
+  py35_docstring:
     docker:
-      - image: circleci/python:2.7
+      - image: circleci/python:3.5
     <<: *set_workdir
     steps:
       - *restore_repo_cache
       - *install_tox
-      - run: tox -e py27-lint_docstring_include_list
+      - run: tox -e py35-lint_docstring_include_list
   py27_first_startup:
     docker:
       - image: circleci/python:2.7
     <<: *set_workdir
     steps:
       - *restore_repo_cache
-      - run: sh scripts/common_startup.sh
-      - run: wget -q https://github.com/jmchilton/galaxy-downloads/raw/master/db_gx_rev_0127.sqlite
-      - run: mv db_gx_rev_0127.sqlite database/universe.sqlite
-      - run: sh manage_db.sh -c ./config/galaxy.yml.sample upgrade
```

Comment:
The idea of leaving this here is to test the latest migrations, as in `.travis.yml` for macOS for py27. Maybe we can move this to py35 for CircleCI or TravisCI to have the full test matrix?

Reply:
Sounds good to me, I'll move this to `py35_first_startup ` in circleci

Reply:
Done in 4066d1bd757732fa7893f62e08e565a0a76d7331